api/v1alpha1/contour_types.go

```diff
@@ -58,6 +42,26 @@ type ContourList struct {
 	Items           []Contour `json:"items"`
 }
 
+// ContourSpec defines the desired state of Contour.
+type ContourSpec struct {
+	// replicas is the desired number of Contour replicas. If unset,
+	// defaults to 2.
+	//
+	// +optional
+	// +kubebuilder:default=2
+	// +kubebuilder:validation:Minimum=0
+	Replicas int32 `json:"replicas,omitempty"`
+}
+
+// ContourStatus defines the observed state of Contour.
+type ContourStatus struct {
+	// availableReplicas is the number of observed available Contour replicas
+	// according to the deployment.
+	//
+	// +kubebuilder:validation:Minimum=0
```

Comment:
Should we really add validation to status (i.e. users don't need validation feedback on this)?

Reply:
Commit `0c03a81` removes the validation for `availableReplicas`.